gensim/scripts/make_wikicorpus.py

```diff
@@ -62,6 +62,12 @@
         print(globals()['__doc__'] % locals())
         sys.exit(1)
     inp, outp = sys.argv[1:3]
+    print "Checking if output directory exists: "+'/'.join(outp.split("/")[:-1])
+    if os.path.isdir('/'.join(outp.split("/")[:-1])) == False:
```

Comment:
I did the changes you suggested. Thank you for explaining. First time contributing to a project!